.github/CODEOWNERS

```diff
@@ -97,11 +97,11 @@ extension/storage/dbstorage/                             @open-telemetry/collect
 extension/storage/filestorage/                           @open-telemetry/collector-contrib-approvers @djaglowski
 
 internal/aws/                                            @open-telemetry/collector-contrib-approvers @Aneurysm9 @mxiamxia
-internal/docker/                                         @open-telemetry/collector-contrib-approvers @mstumpfx @rmfitzpatrick
+internal/docker/                                         @open-telemetry/collector-contrib-approvers @rmfitzpatrick
```

Comment:
Might be worth adding @jamesmoessis here as well as the other code owner of the docker receiver?

Reply:
let's do it :)

Reply:
Happy to :)